pkg/update/update_cockroach_version.go

```diff
@@ -348,13 +321,6 @@ func grantRoleToUser(ctx context.Context, db *sql.DB, role roleMembership, usern
 }
 
 func getClusterSetting(ctx context.Context, db *sql.DB, name string) (string, error) {
-	// TODO do we need this?
-	/*
-		if err := isValidClusterSettingName(name); err != nil {
```

Comment:
Do we need this?? @chrisseto

Reply:
Nope, that's a sanity check from our code base in case the function ever accepts users input.

I think it's safe to say that it won't ever handle user input in this case.